.github/ISSUE_TEMPLATE/4.premium-support.yml

```diff
@@ -1,7 +1,8 @@
 name: 'Premium plan: question ❔'
 description: I'm a Premium plan user, I can't find a solution to my problem with MUI X.
 title: '[question] '
-labels: ['status: needs triage', 'Premium plan', 'support: commercial']
+labels:
+  ['status: needs triage', 'Premium plan', 'support: commercial', 'priority support: unverified']
```

Comment:
We might want to do the opposite, add the verify label because people can open issues without any templates: https://github.com/mui/mui-x/issues/new

Reply:
Plus, assuming that we are pushing in the direction of having the issues validated by a bot, issues would start with no labels, the bot adding them https://github.com/mui/mui-x/issues/6899#issuecomment-1455144356